aiida_quantumespresso/cli/calculations/pw2wannier90.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+"""
+Command line scripts to launch a `Pw2WannierCalculation` for testing and demonstration purposes.
+
+This launcher assumes that the SEED and the PREFIX used in the previous PW calculation (parent_folder)
+are the same as those hardcoded in the Pw2WannierCalculation class.
+We also hardcode some parameters and options.
+"""
+from __future__ import absolute_import
+import click
+
+from aiida.cmdline.params import options, types
+from aiida.cmdline.params.options import OverridableOption
+from aiida.cmdline.utils import decorators
+
+from aiida_quantumespresso.cli.utils import options as options_qe
+
+PARENT_FOLDER = OverridableOption(
+    '-F',
+    '--parent_folder',
+    'parent_folder',
+    metavar='RemoteData/FolderData',  # placeholder in the help text
+    type=types.NodeParamType(sub_classes=('aiida.data:remote', 'aiida.data:folder')),
+    help='A RemoteData or FolderData node identified by its ID or UUID.')
```

Comment:
Just as an FYI, technically the `NodeParamType` also supports loading the node from its label if it has one and it is unique

Reply:
I don't understand what you mean, sorry. What's the label?

Reply:
In AiiDA, each `Node` has three identifiers: its `pk`, `uuid` and `label`. The `label` is the one you can set for example on the `ProcessNode` through the input `metadata.label`, but this is a property of every node. It is simply one of the columns of the node table. Now, since the label is not necessarily unique, unlike the pk and uuid, it might not be very useful to use, but *it can* be used for this. You could for example label a specific `FolderData` with a special label, which you could then use to reference it, instead of memorizing its pk or uuid.

Reply:
Interesting, thanks. I tried it and it works already with this cli launcher.